docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,69 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. The distributor uses rsync over
+ssh to perform the file transfer.
+
+Configuration
+=============
+Here's an example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "my_rpm_rsync_distributor",
+     "distributor_type_id": "rpm_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_user": "foo",
+            "ssh_identity_file": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "yum_distributor",
+        }
+    }
+
+``auth_type``
+  Two authentication methods are supported: ``publickey`` and ``password``.
+
+``ssh_user``
+  ssh login to remote machine
+
+``ssh_identity_file``
+  A path to the private key that will be used as identity file for ssh. When ``auth_type`` is
+  ``publickey`` this is a required config. The key has to be readable by user ``apache``.
+
+``ssh_password``
+  Password is required when ``auth_type`` is 'password'.
+
+``host``
+  remote hostname
+
+``root``
+  Absolute path to the remote root directory where all the data (content and published content)
+  lives. Remote equivalent to ``/var/lib/pulp``.
+
+Optional configuration
+----------------------
+
+``skip_fast_forward``
+  If true, rsync distributor will publish all content of repository.
```

Comment:
I added the 'predistributor_id' to the docs and put some wording about incremental publishes in it's description. I also expanded on the description of this setting.